.github/workflows/master.yml

```diff
@@ -3,22 +3,24 @@ name: Continuous integration
 on:
   push:
     branches:
-      - master
+      - BC-70-ci-cd-github-actions
```

Comment:
please, configure final value of the branch (`development` for now). also make necessary changes in https://github.com/thenewboston-developers/Node/settings/environments/358381439/edit (do we really need to limit branches if already have it configured here?)